dev/sensor/htu21d/export/dev/sensor/htu21d.hpp

```diff
@@ -233,24 +251,14 @@ err htu21d<i2c_dev>::i2c_get_sample_hold_master(uint8_t cmd, uint16_t &sample)
     i2c_dev::platform_handle::set_slave_addr(i2_addr);
 
     // send command to start measuring
-    i2c_dev::lock();
-    err rc = i2c_dev::set_buffers(&cmd, nullptr, 1);
-    if (rc == err::ok) {
-        rc = i2c_dev::xfer();
-    }
-    i2c_dev::unlock();
+    err rc = try_xfer(cmd, nullptr, 1);
 
     if (rc != err::ok) {
         return rc;
     }
 
     // read data, last byte is CRC
-    i2c_dev::lock();
-    rc = i2c_dev::set_buffers(nullptr, data, sizeof(data));
-    if (rc == err::ok) {
-        rc = i2c_dev::xfer();
-    }
-    i2c_dev::unlock();
+    rc = try_xfer(0, data, sizeof(data));
```

Comment:
`0` is a magic number. Add `CMD_NONE` to sensor I2C commands enum and use it instead of `0`